src/SourceLocator/Type/PhpInternalSourceLocator.php

```diff
@@ -38,9 +38,7 @@ protected function createLocatedSource(Identifier $identifier) : ?LocatedSource
              * @todo this code path looks never used, and disagrees with the contract anyway...?
```

Comment:
It is still relevant - I only replaced the type, but it is still dead code that will need fixing later on :-\